### PR TITLE
closes purpleidea/puppet-gluster#36

### PR DESCRIPTION
**Pull request by jackdpeterson**
Add documentation

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -533,6 +533,12 @@ puppet module to be present on the puppet server in the _modules/_ directory.
 The *module_data* code should be in a module folder named: *module_data/*.
 That's it!
 
+###I just upgraded puppet-gluster and my UUIDs keep resetting to 00000000-0000-0000-0000-000000000000
+The following commands may also be failing [gluster pool list, gluster peer status] on some or all of the gluster servers.
+Furthermore, some hosts may see other servers while others list the other peers but remain in a disconnected state.
```
Comment (purpleidea) [inline DOCUMENTATION.md]: s/while others list/while others can list/   ??

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -533,6 +533,12 @@ puppet module to be present on the puppet server in the _modules/_ directory.
...
+
+In one case, this was caused by SourceTree's approach to cloning where it was pulling in all submodules on the Windows OS and/or converting LF to CRLF compared to how a git clone command pulls in the repository on a linux OS. In order to resolve this you must delete the puppet-gluster module directory in its entirety and re-clone it directly on the target puppet master. If you are using version control to save you puppet manifests/modules, make sure that you perform the appropriate addremove commmands and recommit + re-push your code.
```
Comment (purpleidea) [inline DOCUMENTATION.md]: s/save you/save your/

**Comment by purpleidea** [comment]
@jackdpeterson Awesome, thanks! Added two comments, please fix, wrap to correct line length (80), and git commit --amend ...

Thanks!

**Comment by jackdpeterson** [comment]
language enhancements applied ;-)

**Comment by purpleidea** [comment]
@jackdpeterson Great, but the lines in your patch are still super long. They should wrap at 80 chars. Can you fix and amend one more time please?

**Comment by jackdpeterson** [comment]
https://github.com/jackdpeterson/puppet-gluster/commit/b8ac93c4b79b87395c43769ef065a81b3003748c should all be wrapped to <= 80 characters in documentation. perhaps github was a bit lagged as I see the notification on github came in after your comment.

**Comment by purpleidea** [comment]
@jackdpeterson Okay, your lines now have an extra space on the end. Git diff --check should catch this for you. Please remove those and rebase.

Now that you've split the topic, it doesn't render correctly. Keep the 000000 line on the single line.

Lastly you should probably replace the [ & ] with ( & ) and enclose the individual commands in ` (backticks) so it displays as commands.

I do thank you for your patch-- I'm not trying to nitpick, but to hopefully help you get used to the need to be detail oriented in your future projects. If you're fed up, LMK and I'll just add the fixes myself. If you do want to continue, please do!

Cheers,
James

PS: remember to squash and rebase against and up to date master!

**Comment by jackdpeterson** [comment]
Okay, latest changes to documentation should be pushed in. I'm unsure about the & comment; however, I modified the commands to fit inside the backticks and it looks okay on github's markdown to HTML viewer. Just pending 37f175d to finish w/ travis.

**Comment by purpleidea** [comment]
@jackdpeterson There was still a bit of formatting to do, and one git diff --check bug. Thanks for this patch. I'm going to just merge it, and add a few changes on top.

Cheers!
James

**Comment by purpleidea** [comment]
(Merged!)

**Pull request closed without merging**